app/views/staff/shared/transfers/_index.html.haml

```diff
@@ -39,6 +41,7 @@
                 %td.govuk-table__cell= transfer.source.organisation.name
                 %td.govuk-table__cell= transfer.source.roda_identifier
               %td.govuk-table__cell= transfer.value
+              %td.govuk-table__cell= transfer.beis_identifier
```

Comment:
I'm not sure of the value of showing this in the table, given it's an optional field, and only useful for historic transfers. I may be wrong though (I frequently am!)

Reply:
The reason I added the historic identifier to the table is that there is no individual transfer show page, so the only other way a user could remind themself what they entered would be to go into edit mode, which might not be intuitive to them.

Reply:
Oh course!